assets/styles/styles.css

```diff
@@ -71,3 +71,10 @@ body {
 .list-group-item ul {
   padding-left:0;
 }
+
+.checkbox input[type=checkbox],
+.checkbox-inline input[type=checkbox],
+.radio input[type=radio],
+.radio-inline input[type=radio] {
+  margin-left: 0;
+}
```

Comment:
Bootstrap shifts this alignment by default expecting nested labels. For 508 compliance, we prefer labels with `for` elements instead, so this adjusts the default spacing.